### ISSUE TITLE
Browser Exec Issue

### ISSUE THREAD

**Issue by Kai2PointOh** [roju/tiktok-live-recorder#14]
```
2023-10-10 00:55:03,899 - INFO - Waiting 1 minute
2023-10-10 00:56:04,793 - INFO - keke8662 is live
2023-10-10 00:56:04,793 - INFO - Getting live url for room ID 7288178700787485482
2023-10-10 00:56:05,088 - INFO - Account is private, login required
2023-10-10 00:56:05,088 - ERROR - BrowserExtractor.__init__() takes 1 positional argument but 3 were given
2023-10-10 00:56:05,089 - INFO - Waiting 1 minute
```

Browser Extension is installed. Command to run this autorecorder is

```python ttlr.py -user keke8662 -mode auto -ffmpeg -browser_exec "C:\Program Files\Google\Chrome\Application\chrome.exe" -out_dir Z:\Repos\lives\kib221```

**Comment by roju** [comment]
The browser_exec feature has not been tested on Windows yet. That may be the cause of your error.

**Comment by roju** [comment]
Should be fixed in v4.2
